cookbook/console/command_in_controller.rst

```diff
@@ -0,0 +1,96 @@
+.. index::
+   single: Console; Use commands in your Controller
+
+How to Call a Command from a Controller
+=======================================
+
+The :doc:`Console component documentation </components/console/introduction>`_ covers
+how to create a console command. This cookbook article covers how to use a console command
+directly in your application. 
+
+You may have the need to execute some function that is only available in a console command. 
+Usually, you should refactor the command and move some logic into a service that can be 
+reused in the controller. However, when the command is part of a third-party library, you 
+wouldn't want to modify or duplicate their code, but want to directly execute the command 
+instead. 
+
+An example of this is sending the emails that Swift Mailer spooled earlier
+:doc:`using the ``swiftmailer:spool:send`` command </cookbook/email/spool>`_. Symfony 
```

Comment:
Will the ``swiftmailer:spool:send`` part work here? Or will this mess up the link?